README.md

```diff
@@ -126,6 +126,44 @@ Follow these steps to set up and run the project using Docker.
 
 You can then access the admin at `http://0.0.0.0:8000/admin/` or `http://localhost:8000/admin/`.
 
+#### Running Wagtail locally with supporting services in Docker
+
+You can also run the main application locally with the supporting backend services such as the Postgres and Redis running in Docker. This can be useful when you want to make changes that require the app to be frequently restarted to be picked up.
+
+In order to run it:
+
+1. Pull the images of the supporting services.
+
+```
+make docker-dev-pull
+```
+
+2. Start the supporting services in Docker.
+
+```
+make docker-dev-up
+```
+
+3. Run the following pre-run steps:
+
+```
+make makemigrations
+make collectstatic
+make migrate
+make createsuperuser
+```
+
+4. Run the Django server locally via your IDE or with the following command:
+
+```
+make runserver
+```
+
+You can specify the runtime configuration either in your IDE (for PyCharm see [here](https://www.jetbrains.com/help/pycharm/run-debug-configuration.html#createExplicitly)), or copy the `.env-dev-example` and rename it to `.env` which will allow Django to pick up the config.
+
+> [!NOTE]
+> When running the application in a virtual environment via Poetry the `.env` file will not be picked up automatically. For this to work you'll need to install the [poetry-plugin-dotenv](https://github.com/pivoshenko/poetry-plugin-dotenv). However if you installed Poetry with `brew` rather than `pip` that currently isn't going to work (see the [issue](https://github.com/pivoshenko/poetry-plugin-dotenv/issues/327)) and you'll need to install an older and seemingly no longer maintained [poetry-dotenv-plugin](https://github.com/mpeteuil/poetry-dotenv-plugin).
```

Comment:
Nit: This maybe helpful for other devs. Maybe you can add a line about this in the `README` file.

I have installed Poetry using [the official installer](https://python-poetry.org/docs/#installing-with-the-official-installer). I think the recommended way is Pipx or using the official installer.

Not sure installing Poetry using pip is a good idea. https://github.com/orgs/python-poetry/discussions/8687

Since I used the official installer, I didn't have to do any of the steps in the `README` =)

Reply:
Also wouldn't this be a dev dependency? If we truly need this, which I didn't need due to installing it using the official installer.